src/routes/profile-route/profile-sidebar.jsx

```diff
@@ -48,6 +49,19 @@ export const ProfileSidebar = () => {
               </div>
             );
           })}
+          <div className="flex">
+            <img
+              src="/src/images/sidebar-arrow.svg"
+              alt="sidebar arrow"
+              className="h-8 w-8"
+            ></img>
+
+            <button onClick={handleLogout} className="gap-5 ml-5">
+              <span className="font-arialBold text-lg hover:border-b-[3px] hover:border-black">
+                Logout
+              </span>
+            </button>
+          </div>
```

Comment:
shouldn't the sidebar arrow be added to the image as well?
with is setup nothing will happen if you click on the arrow image right?

Reply:
for now, this is fine how it is. making the icons clickable would require more refactoring than is within the scope of this ticket. we can create another ticket later on if we want to make the icons clickable.

Reply:
There's a pretty simple solution that works, although it leaves the gap between the button and text clickable:

Moving the image inside of the button element will include it as part of the button. You'll need to add flex and flex-row to align them however.

```
 <button onClick={handleLogout} className="gap-5 flex flex-row">
              <img
                src="/src/images/sidebar-arrow.svg"
                alt="sidebar arrow"
                className="h-8 w-8"
              ></img>

              <span className="font-arialBold text-lg hover:border-b-[3px] hover:border-black">
                Logout
              </span>
            </button>
```